packages/api/src/routers/account-recovery/router.ts

```diff
@@ -0,0 +1,265 @@
+import { TRPCError } from '@trpc/server'
+import debug from 'debug'
+import { SUPABASE_SUBDOMAIN } from 'app/utils/supabase/admin'
+import { createTRPCRouter, publicProcedure } from '../../trpc'
+import { z } from 'zod'
+import {
+  type ChallengeResponse,
+  type VerifyChallengeResponse,
+  RecoveryOptions,
+} from '@my/api/src/routers/account-recovery/types'
+import {
+  getChallengeById,
+  getChainAddress,
+  getPasskey,
+  isChallengeExpired,
+  insertChallenge,
+} from 'app/utils/account-recovery'
+import { mintAuthenticatedJWTToken } from 'app/utils/jwt'
+import { verifyMessage, hexToBytes } from 'viem'
+import { verifySignature } from 'app/utils/userop'
+import { COSEECDHAtoXY } from 'app/utils/passkeys'
+import { pgBase16ToHex } from 'app/utils/pgBase16ToHex'
+
+const logger = debug('api:routers:account-recovery')
+
+export const GetChallengeResponseSchema = z.object({
```

Comment:
no need for the other fields.

```suggestion
export const GetChallengeResponseSchema = z.object({
  challenge: z.string(),
  expires_at: z.string(),
})
```